test-packages/release/src/publish.ts

```diff
@@ -116,15 +208,20 @@ To publish a release you should start from a clean repo. Run "embroider-release
   // the end.
   let reporter = new IssueReporter();
 
-  await makeTags(solution, reporter);
-  await push(reporter);
-  await createGithubRelease(octokit, description, representativeTag, reporter);
-  await pnpmPublish(solution, reporter);
+  await makeTags(solution, reporter, dryRun);
+  await push(reporter, dryRun);
+  await createGithubRelease(octokit, description, representativeTag, reporter, dryRun);
+  await pnpmPublish(solution, reporter, dryRun);
```

Comment:
Should the order be `makeTags`, `pnpmPublish`, `pushTags`, `createGithubRelease`?

The `pnpmPublish` is more consequential, if it failed it's probably better to leave the unpushed

Reply:
It wasn't my intent to change the semantics or change the order of things in this PR, but considering how making the release automated requires extra care, I think this makes sense -- will update.

Reply:
Will def want @ef4's opinion here tho

Reply:
Seem good to me.